src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -149,6 +151,7 @@ def __str__(self):
         active_directory='https://login.microsoftonline.com',
         active_directory_resource_id='https://management.core.windows.net/',
         active_directory_graph_resource_id='https://graph.windows.net/',
+        active_directory_data_lake_resource_id='https://datalake.azure.net/',
```

Comment:
You are correct, we are only supported in the public cloud for right now. As we on board into other clouds we will add the correct resource_id for that cloud.